features/republican_silence.feature

```diff
@@ -0,0 +1,54 @@
+Feature:
+  As Referent|CP-Host|Committee-Host or Committee-Supervisor
+  I cannot communicate with the adherents when one republican silence is declared for the same Referent Tags
+
+  Background:
+    Given the following fixtures are loaded:
+      | LoadCitizenActionData     |
+      | LoadRepublicanSilenceData |
+
+  Scenario Outline: As referent of department 92 I cannot communicate with adherent from my referent space.
+    Given I am logged as "referent@en-marche-dev.fr"
+     When I go to "<uri>"
+     Then I should see "En raison du silence républicain, votre espace est momentanément désactivé. Vous pourrez de nouveau y accéder à la fin de celui-ci."
+    Examples:
+      | uri                                   |
+      | /espace-referent/utilisateurs         |
+      | /espace-referent/utilisateurs/message |
+      | /espace-referent/evenements/creer     |
+
+  Scenario Outline: As committee host I cannot access to the committee pages
+    Given I am logged as "lolodie.dutemps@hotnix.tld"
+    When I go to "<uri>"
+    Then I should see "En raison du silence républicain, votre espace est momentanément désactivé. Vous pourrez de nouveau y accéder à la fin de celui-ci."
+    Examples:
+      | uri                                                       |
+      | /comites/en-marche-comite-de-singapour                    |
+      | /comites/en-marche-comite-de-singapour/evenements/ajouter |
+
+  @javascript
```

Comment:
yes, I don't know why but the test not passed without

Reply:
Perhaps because the HTML structure is not correct for BrowserKit.

It's worth it to try to fix this issue as Selenium tests are slow.

My guess is that ```And I check "members[]"``` is not withing a `<form>`.